grafana/knative-reconciler.json

```diff
@@ -0,0 +1,684 @@
+				"values": false
+			},
+			"lines": true,
+			"linewidth": 1,
+			"links": [],
+			"nullPointMode": "null",
+			"options": {
+				"alertThreshold": true
+			},
+			"percentage": false,
+			"pluginVersion": "8.0.1",
+			"pointradius": 5,
+			"points": false,
+			"renderer": "flot",
+			"seriesOverrides": [],
+			"spaceLength": 10,
+			"stack": false,
+			"steppedLine": true,
+			"targets": [{
+					"expr": "histogram_quantile(0.99, sum(rate(controller_reconcile_latency_bucket{reconciler=\"$reconciler\", key=\"$key\"}[1m])) by (le))",
+					"format": "time_series",
+					"instant": false,
+					"interval": "",
+					"intervalFactor": 1,
+					"legendFormat": "99th",
+					"refId": "A"
+				},
+				{
+					"expr": "histogram_quantile(0.90, sum(rate(controller_reconcile_latency_bucket{reconciler=\"$reconciler\", key=\"$key\"}[1m])) by (le))",
+					"format": "time_series",
+					"intervalFactor": 1,
+					"legendFormat": "90th",
+					"refId": "B"
+				},
+				{
+					"expr": "histogram_quantile(0.50, sum(rate(controller_reconcile_latency_bucket{reconciler=\"$reconciler\", key=\"$key\"}[1m])) by (le))",
+					"format": "time_series",
+					"intervalFactor": 1,
+					"legendFormat": "50th",
+					"refId": "C"
+				}
+			],
+			"thresholds": [],
+			"timeFrom": null,
+			"timeRegions": [],
+			"timeShift": null,
+			"title": "$reconciler/$key Reconcile Latency Percentiles",
+			"tooltip": {
+				"shared": true,
+				"sort": 0,
+				"value_type": "individual"
+			},
+			"type": "graph",
+			"xaxis": {
+				"buckets": null,
+				"mode": "time",
+				"name": null,
+				"show": true,
+				"values": []
+			},
+			"yaxes": [{
+					"format": "short",
+					"label": null,
+					"logBase": 1,
+					"max": null,
+					"min": null,
+					"show": true
+				},
+				{
+					"format": "short",
+					"label": null,
+					"logBase": 1,
+					"max": null,
+					"min": null,
+					"show": true
+				}
+			],
+			"yaxis": {
+				"align": false,
+				"alignLevel": null
+			}
+		}
+	],
+	"refresh": "5s",
+	"schemaVersion": 30,
+	"style": "dark",
+	"tags": [
+		"Knative"
+	],
+	"templating": {
+		"list": [{
+				"allValue": null,
+				"current": {},
+				"datasource": "${DS_PROMETHEUS}",
+				"definition": "label_values(controller_reconcile_count, reconciler)",
+				"description": null,
+				"error": null,
+				"hide": 0,
+				"includeAll": false,
+				"label": "Reconciler",
+				"multi": false,
+				"name": "reconciler",
+				"options": [],
+				"query": {
+					"query": "label_values(controller_reconcile_count, reconciler)",
+					"refId": "StandardVariableQuery"
+				},
+				"refresh": 1,
+				"regex": "",
+				"skipUrlSync": false,
+				"sort": 0,
+				"tagValuesQuery": "",
+				"tagsQuery": "",
+				"type": "query",
+				"useTags": false
+			},
+			{
+				"allValue": null,
+				"current": {},
+				"datasource": "${DS_PROMETHEUS}",
+				"definition": "label_values(controller_reconcile_count{reconciler=\"$reconciler\"}, key)",
+				"description": null,
```

Comment:
`Key` does not exist in currently emitted metrics.
Here is a sample of what controller emits:
```
controller_reconcile_count{namespace_name="default",reconciler="knative.dev.serving.pkg.reconciler.gc.reconciler",success="true"} 11
```